plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -100,4 +100,207 @@
             id="com.google.cloud.tools.eclipse.gcloud.runtime">
       </serverdefinition>
    </extension>
+   
+   <extension
+         point="org.eclipse.wst.server.core.runtimeTypes">
+      <runtimeType
+            class="com.google.cloud.tools.eclipse.appengine.localserver.runtime.CloudSdkRuntime"
+            description="%runtimeTypeDescription"
+            id="com.google.cloud.tools.eclipse.runtime.id1"
+            name="%runtimeTypeName"
+            vendor="%runtimeTypeVendor"
+            version="1">
+            <moduleType types="jst.web" versions="2.2, 2.3, 2.4, 2.5, 3.1"/>
```

Comment:
@briandealwis It does not support the full 3.1 spec, but it supports annotations, so my thinking here was to allow for 3.1 so Eclipse does not warn about annotations. But I agree, that users might be tempted to use other parts of the spec that is not supported by AE Standard Environment to find it out only later, and possibly not in a very straightforward way.

I'd keep the code as is, but file an issue to find a solution later.